task/jenkins/0.1/README.md

```diff
@@ -0,0 +1,113 @@
+# Jenkins Task
+
+The following task can be used to interact with Jenkins with Jenkins using the Jenkins REST API.
+
+More details on Remote Access API can be found [here](https://www.jenkins.io/doc/book/using/remote-access-api/)
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/jenkins/0.1/jenkins.yaml
+```
+
+## Parameters
+
+- **JENKINS_HOST_URL**: The URL on which Jenkins is running (**Required**)
+- **JENKINS_SECRETS**: The name of the secret containing the username and API token for authenticating the Jenkins (_Default_: jenkins-credentials) (**Required**)
+- **ARGS**: Extra arguments to add to the control script. (**Required**)
+
+The arguments are :
+
+`start`: Start a new Jenkins job.
+
```

Comment:
can we use the numbered list so that it is a bit more clearer. example
1. `start`: Start a new Jenkins job